src/actions/registerPanel.ts

```diff
@@ -9,26 +9,32 @@ export interface EchoPanelRegister {
 }
 
 export interface EchoPanelOptions {
-    searchActive?: boolean;
-    customPanelActive?: string;
+    searchActive: boolean;
+    customPanelActive: string;
     addSearch: boolean;
+    paddingTop: number;
```

Comment:
ta inn et styling object (CssProperties) i stedet for paddingTop så det ikke er så begrensende?

Reply:
hvis vi ønsker å gjøre det må vi si at pannel UI skal vera definert slik. 

`
interface pannelUI {
[key: string]: CssProperties
}
`

da vil vi kunne si at pannelUI skal kunne  være  ekx.

const ui: panelUI = {
 pannelTo: {top: "65px"}
} 

det er absolut en mulighet

Reply:
mulig du skal bruke så kan den brukes flere plasser :)
type UI = { [key: string]: CssProperties }

Reply:
Så panelUI skal ikke bare brukes på Panelwrapper div-en
Men man skal definere hvor i panelet den ui-en skal legges til?

Reply:
Tanken er å ha et object vi kanlegge UI endringer som moduler skulle trenge. f.eks flytting av legend, panel with osv...

Reply:
Skjønner, derav { [key: string]: CssProperties }. Da gir det mening